src/upload.test.ts

```diff
@@ -341,10 +351,58 @@ describe("splitSizeIntoChunkAlignedParts", () => {
         { start: 400 * mib, end: 500 * mib },
       ],
     ],
+
+    // Use larger chunk size.
+    [
+      40 * mib,
+      2,
+      chunk * 2,
+      [
+        { start: 0, end: 0 },
+        { start: 0, end: 40 * mib },
+      ],
+    ],
+    [
+      40 * mib,
+      3,
+      chunk * 3,
+      [
+        { start: 0, end: 0 },
+        { start: 0, end: 0 },
+        { start: 0, end: 40 * mib },
+      ],
+    ],
+    [
+      41 * mib,
+      2,
+      chunk * 2,
+      [
+        { start: 0, end: 0 },
+        { start: 0, end: 41 * mib },
+      ],
+    ],
+    [
+      80 * mib,
+      2,
+      chunk * 2,
+      [
+        { start: 0, end: 80 * mib },
+        { start: 80 * mib, end: 80 * mib },
+      ],
+    ],
+    [
+      81 * mib,
+      2,
+      chunk * 2,
+      [
+        { start: 0, end: 80 * mib },
+        { start: 80 * mib, end: 81 * mib },
+      ],
+    ],
   ];
 
-  it.each(sizesAndChunks)("Should align size '%s' with '%s' parts", (totalSize, partCount, expectedParts) => {
-    const parts = splitSizeIntoChunkAlignedParts(totalSize, partCount);
+  it.each(sizesAndChunks)("('%s', '%s', '%s')", (totalSize, partCount, chunkSize, expectedParts) => {
```

Comment:
I think the text test name was nice, why did you remove it in favour of just listing the arguments?

Reply:
I don't know what the standard practice is, but I had a few reasons for doing it this way. One, I thought it made it more clear what arguments the function was being called with. The test would have the name `splitSizeIntoChunkAlignedParts > (arguments)` which makes it easy to parse visually. I thought the more descriptive name was messier, and sometimes it was awkward to try to word it in a way such that the arguments appear in order in the sentence.

That's just my own reasoning though and I have no idea what the best practice is. I've struggled with naming the tests until I just started listing the arguments and it made it simpler and faster. 🤷‍♂️ Thoughts?

Reply:
I pushed an attempt at making it a little better, let me know what you think.

Reply:
My 2c:
Test names are nice because they give you some immediate context when you see them in a log. They don't need to be semantically perfect or anything, just something that will give you an initial clue and it's easy to grep for, e.g. "Uploads should align to chunk size".

Reply:
> I pushed an attempt at making it a little better, let me know what you think.

Yeah that's a bit better, thanks 👍